src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,77 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private final RegistrationServiceImpl registrationService = new RegistrationServiceImpl();
+    private User user;
+
+    @BeforeEach
+    void setUp() {
+        user = new User();
+        user.setLogin("Bob");
+        user.setAge(20);
+        user.setPassword("password");
+    }
+
+    @AfterEach
+    void clearAfterEach() {
+        Storage.people.clear();
+    }
+
+    @Test
+    void register_nullLogin_NotOk() {
+        user = null;
+        assertThrows(NullPointerException.class, () -> {
+            registrationService.register(user);
+        });
+    }

```

Comment:
here you check if all object user is null. you need to create separate method that will check all separate fields in user object on null and method that will check all object user on null